packages/office-addin-manifest/test/manifests/manifest.json

```diff
@@ -1,120 +1,82 @@
 {
   "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
-  "id": "df2fa15f-332a-47aa-87c2-e30c20686eee",
-  "manifestVersion": "m365DevPreview",
+  "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee",
   "version": "1.0.0",
+  "manifestVersion": "m365DevPreview",
   "name": {
-    "short": "My Office Add-in",
-    "full": "My Office Add-in"
+    "short": "Contoso Task Pane Add-in",
+    "full": "Contoso Task Pane Add-in"
   },
   "description": {
     "short": "A template to get started.",
-    "full": "A template to get started."
+    "full": "This is the template to get started."
   },
-  "icons": {
-    "outline": "test/assets/icon-32.png",
-    "color": "test/assets/icon-32.png"
-  },
-  "accentColor": "#FFFFFF",
   "developer": {
     "name": "Contoso",
     "websiteUrl": "https://www.contoso.com",
     "privacyUrl": "https://www.contoso.com/privacy",
-    "termsOfUseUrl": "https://www.contoso.com/terms"
+    "termsOfUseUrl": "https://www.contoso.com/servicesagreement"
+  },
+  "icons": {
```

Comment:
The conversion tool may require this to be a full url (i.e. using localhost:3000 as the domain)

Reply:
All the examples I saw used https URLs.
Might be just for examples, but we can change these if we found out it later.

Reply:
If the examples are using https URLs then we should to.

Reply:
Use:

    "icons": {
      "outline": "https://localhost:3000/assets/icon-64.png",
      "color": "https://localhost:3000/assets/icon-128.png"
    },